sympy/printing/pretty/pretty.py

```diff
@@ -1522,6 +1521,30 @@ def _print_Contains(self, e):
         else:
             return prettyForm(sstr(e))
 
+    def _print_SeqFormula(self, s):
```

Comment:
Well from user point of view it should never occur. But from a developer point of view, it will become a headache if everytime you initialize ``SeqExpr`` in isympy shell and you get a big error. I did have the case, when I had to initialize it, for argument checking. So, I think it's not a good idea.